src/HttpReply.h

```diff
@@ -82,6 +82,9 @@ class HttpReply: public HttpMsg
     /** \return a ready to use mem buffer with a packed reply */
     MemBuf *pack() const;
 
+    /// construct and return a 200 (Connection established)
+    static HttpReply *makeConnectionEstablished();
```

Comment:
We should capitalize static methods:

```suggestion
    static HttpReply *MakeConnectionEstablished();
```